dist_app.tcjs

```diff
@@ -10,7 +9,6 @@ tc.prerequisites = [
 ];
 tc.compilationMakeType = MakeType.MS_nmake;
 tc.compileArguments = '$(DEBUG_TAG)';
-tc.linkArguments = '/DEBUG /MACHINE:X64 /LIBPATH:"' + tc.pocoLoc + '\\lib64"';
```

Comment:
/DEBUG /MACHINE:X64 was removed from linkArguments, was that intended?

Reply:
Yes, because we are not linking with poco lib.